terminus-ui/src/select/select.component.ts

```diff
@@ -685,6 +687,30 @@ export class TsSelectComponent implements
   }
   private _delimiter: string = DEFAULT_DELIMITER;
 
+  /**
+   * Define a function to retrieve the UI value for an option
+   */
+  @Input()
+  public set formatUIFn(value: TsSelectFormatFn) {
```

Comment:
Can we rename this to something more explicit? Maybe `chipFormatFn` or `chipFormatUIFn` etc just so it's clear this will only affect chips and not all selects.